astropy/units/format/base.py

```diff
@@ -152,6 +152,10 @@ def to_string(
         ValueError
             If ``fraction`` is not recognized.
         """
+        return cls._to_string(unit, fraction=fraction)
```

Comment:
It might be worth having a comment here on why we separate the methods. At some level, I don't think `Base.to_string()` is still used at all, so perhaps it should be removed?

Reply:
`UnitBase.to_string()` calls the `to_string()` method of the specified unit formatter. A public `Base.to_string()` method declares to downstream developers that this is something a unit formatter must have and also provides the default implementation, including to our own `Generic`.